frontend/src/app/grants/AbstractDialog.tsx

```diff
@@ -51,7 +52,12 @@ const AbstractDialog = () => {
       {grant && (
         <>
           <DialogTitle>
-            <Title variant='h5'>{grant.title}</Title>
+            <Title variant='h5'>
+              {grant.title}
+              {grant.external_url ? (
+                <a href={grant.external_url} target="_blank" rel="noreferrer"><LaunchIcon /></a>
+              ) : null}
```

Comment:
```suggestion
              {grant.external_url && (
                <a href={grant.external_url} target="_blank" rel="noreferrer"><LaunchIcon /></a>
              )}
```
This is how I usually express this type of JSX conditional logic